### DISCUSSION TITLE
Azure Data Studio asking to save changes?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/azuredatastudio#23118] [Q&A]
Hi,
I'm used to have Azure Data Studio keeping open files open no matter I close it, even unsaved ones. Now, in one of my computers it just started asking if I want to save them, and if I don't they're gone. Hot files is onExit, every other setting looks the same as the other computers.
Any clue of what may have changed? Is one of the features I loved the most.
![image](https://github.com/microsoft/azuredatastudio/assets/35255516/93089069-a8bd-4544-9c07-01104cc86c51)

**Comment by erinstellato-ms** [comment]
@CarlosJSA Can you confirm what version you are running? (screenshot of About screen is great)

**Comment by ghost** [comment]
Hi Erin,

Apologies I didn't specify. I'm running latest stable version, and the same version in other computers without issues:

![image](https://github.com/microsoft/azuredatastudio/assets/35255516/43e21ac6-443c-4ef5-8ddd-d77634b907fa)

**Comment by erinstellato-ms** [comment]
@CarlosJSA No worries.  I am not able to replicate this here.  What extensions do you have installed?  It's possible (?) it could be related to [this issue](https://github.com/microsoft/azuredatastudio/discussions/23118) with workspaces...are you using those?

**Comment by ghost** [comment]
Hi Erin,

Thanks for your answers. When I checked today, is working and I can't reproduce the issue anymore. I haven't changed absolutely anything, and though I do have a couple of basic extensions they haven't been updated either and they are the same ones I have in other computers. 

I guess I'll never know, but it works now. :)

**Comment by erinstellato-ms** [comment]
@CarlosJSA Thanks for the update!  I'll close this for now, hopefully it was a very strange one-off scenario and won't be an issue moving forward.
